plio/tests.py

```diff
@@ -135,3 +135,106 @@ def setUp(self):
     def test_for_question(self):
         # write API calls here
         self.assertTrue(True)
+
+
+class ImageTestCase(PlioTestCase):
+    """Tests the Image CRUD functionality."""
+
+    def setUp(self):
+        # seed some plios
+        super().setUp()
+
+        # seed some items and questions
+        self.plio_1 = Plio.objects.filter(name="Plio 1").first()
```

Comment:
is it sure that this test case will run before PlioTestCase? why depend on it instead of creating a new plio instance here itself?